src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/items/BaseSTBBlock.java

```diff
@@ -509,7 +509,8 @@ public Block getAuxiliaryBlock(RelativePosition pos) {
 
     public Block getAuxiliaryBlock(Location loc, RelativePosition pos) {
         Block b = loc.getBlock();
-        int dx = 0, dz = 0;
+        int dx = 0,
+            dz = 0;
```

Comment:
Variables should never be declared in pairs, seperate these declarations please.
```suggestion
        int dx = 0;
        int dz = 0;
```